### PR TITLE
My footer

### PR DESCRIPTION
**Pull request by LeonidSTP**

### CODE REVIEW

File: .angular2cli/src/app/components/MyFooter/myfooter.html [hunk 1]
```diff
@@ -1,7 +1,6 @@
-
-  <div class="panel panel-default">
-  <div class="panel-body">
-    Panel content
-  </div>
-  <div class="panel-footer">Panel footer</div>
+<footer class="footer">
+  <div class="container-fluid">
+    <div class="panel-footer">&copy; Fabulous messenger 2017</div>
```
Comment (pogodich) [inline .angular2cli/src/app/components/MyFooter/myfooter.html]: use js class Date for get current year

File: .angular2cli/src/app/app.module.ts [hunk 2]
```diff
@@ -12,7 +13,8 @@ const appRoutes: Routes = [
 @NgModule({
     declarations: [
         AppComponent,
-        AuthFormComponent
+        AuthFormComponent,
+      MyfooterComponent
```
Comment (redpanda8814) [inline .angular2cli/src/app/app.module.ts]: Keep formatting in the same style

File: .angular2cli/src/app/components/MyFooter/myfooter.html [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<footer class="footer">
+  <div class="container-fluid">
+    <p class="m-0 panel-footer">&copy; Fabulous messenger 2017</p>
```
Comment (redpanda8814) [inline .angular2cli/src/app/components/MyFooter/myfooter.html]: Use Date here, to dynamically change year in future

File: .angular2cli/src/app/components/MyFooter/myfooter.component.ts [hunk 4]
```diff
@@ -6,6 +6,8 @@ import {Component, OnInit} from '@angular/core';
   templateUrl: 'myfooter.html'
 })
 export class MyfooterComponent implements OnInit{
+  public year = new Date().getFullYear();
+  public currentYear: number = this.year;
   ngOnInit(){
     console.log('234');
```
Comment (pogodich) [inline .angular2cli/src/app/components/MyFooter/myfooter.component.ts]: remove console

File: .angular2cli/src/app/components/MyFooter/myfooter.component.ts [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+@Component({
+  selector: 'myfooter',
+  styleUrls: ['myfooter.less'],
+  templateUrl: 'myfooter.html'
+})
+export class MyfooterComponent implements OnInit{
+  public year = new Date().getFullYear();
+  public currentYear: number = this.year;
+  ngOnInit(){
+    console.log('234');
```
Comment (pogodich) [inline .angular2cli/src/app/components/MyFooter/myfooter.component.ts]: remove console

File: .angular2cli/src/app/components/MyFooter/myfooter.component.ts [hunk 6]
```diff
@@ -0,0 +1,14 @@
+import {Component, OnInit} from '@angular/core';
+
+@Component({
+  selector: 'myfooter',
+  styleUrls: ['myfooter.less'],
+  templateUrl: 'myfooter.html'
+})
+export class MyfooterComponent implements OnInit{
+  public year = new Date().getFullYear();
+  public currentYear: number = this.year;
```
Comment (pogodich) [inline .angular2cli/src/app/components/MyFooter/myfooter.component.ts]: useless variable year
public currentYear: number = new Date().getFullYear();

**Pull request merged**